From b2296b20a1abf8f3753fbddef97f4831a292409c Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sun, 4 May 2014 11:23:17 -0700
Subject: [PATCH:libX11] XCreateGC.man: simplify table to work with Solaris
 tbl

Having every table cell be a text diversion (T{...T}) was too much for
Solaris tbl to handle, and thus "man XCreateGC" would print the error
/usr/man/man3x11/XCreateGC.3x11: line 402: Too many text block diversions
tbl quits
and not display the table of mask bits or any text in the man page after
that table.   Since the #define column doesn't need special handling,
making it not use text diversions brings the table under the tbl limit.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 man/XCreateGC.man |   94 ++++++++++++++---------------------------------------
 1 file changed, 24 insertions(+), 70 deletions(-)

diff --git a/man/XCreateGC.man b/man/XCreateGC.man
index de0d039..5f021a7 100644
--- a/man/XCreateGC.man
+++ b/man/XCreateGC.man
@@ -247,173 +247,127 @@ The
 .ZN XGCValues
 structure contains:
 .LP
-.LP
 /\&* GC attribute value mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(.75i).
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCFunction
 T}	T{
 (1L<<0)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCPlaneMask
 T}	T{
 (1L<<1)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCForeground
 T}	T{
 (1L<<2)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCBackground
 T}	T{
 (1L<<3)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCLineWidth
 T}	T{
 (1L<<4)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCLineStyle
 T}	T{
 (1L<<5)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCCapStyle
 T}	T{
 (1L<<6)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCJoinStyle
 T}	T{
 (1L<<7)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCFillStyle
 T}	T{
 (1L<<8)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCFillRule
 T}	T{
 (1L<<9)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCTile
 T}	T{
 (1L<<10)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCStipple
 T}	T{
 (1L<<11)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCTileStipXOrigin
 T}	T{
 (1L<<12)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCTileStipYOrigin
 T}	T{
 (1L<<13)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCFont
 T}	T{
 (1L<<14)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCSubwindowMode
 T}	T{
 (1L<<15)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCGraphicsExposures
 T}	T{
 (1L<<16)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCClipXOrigin
 T}	T{
 (1L<<17)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCClipYOrigin
 T}	T{
 (1L<<18)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCClipMask
 T}	T{
 (1L<<19)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCDashOffset
 T}	T{
 (1L<<20)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCDashList
 T}	T{
 (1L<<21)
 T}
-T{
-\&#define
-T}	T{
+\&#define	T{
 .ZN GCArcMode
 T}	T{
 (1L<<22)
 T}
 .TE
 .IN "XGCValues" "" "@DEF@"
+.LP
 .Ds 0
 /\&* Values */
 
-- 
1.7.9.2

From 6575a906e98888a13d772692ca9df4dd01d153cf Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sun, 4 May 2014 11:54:59 -0700
Subject: [PATCH:libX11] Add missing .TE tags to end tables in Xkb man pages

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 man/xkb/XkbSetIgnoreLockMods.man     |    1 +
 man/xkb/XkbSetServerInternalMods.man |    1 +
 2 files changed, 2 insertions(+)

diff --git a/man/xkb/XkbSetIgnoreLockMods.man b/man/xkb/XkbSetIgnoreLockMods.man
index e3dd33d..635c7ba 100644
--- a/man/xkb/XkbSetIgnoreLockMods.man
+++ b/man/xkb/XkbSetIgnoreLockMods.man
@@ -187,6 +187,7 @@ XkbNamesMask	names	(1L<<4)
 XkbCompatMapMask	compat	(1L<<5)
 XkbGeometryMask	geom	(1L<<6)
 XkbAllComponentsMask	All Fields	(0x7f)
+.TE
 .SH "SEE ALSO"
 .BR XkbGetControls (__libmansuffix__),
 .BR XkbGetNames (__libmansuffix__),
diff --git a/man/xkb/XkbSetServerInternalMods.man b/man/xkb/XkbSetServerInternalMods.man
index 93ede9b..666e59f 100644
--- a/man/xkb/XkbSetServerInternalMods.man
+++ b/man/xkb/XkbSetServerInternalMods.man
@@ -191,6 +191,7 @@ XkbNamesMask	names	(1L<<4)
 XkbCompatMapMask	compat	(1L<<5)
 XkbGeometryMask	geom	(1L<<6)
 XkbAllComponentsMask	All Fields	(0x7f)
+.TE
 .SH "SEE ALSO"
 .BR XkbGetControls (__libmansuffix__),
 .BR XkbGetNames (__libmansuffix__),
-- 
1.7.9.2

